src/applications/appeals/996/config/submitForm.js

```diff
@@ -21,7 +19,7 @@ const submitForm = (form, formConfig) => {
     ? formConfig.transformForSubmit(formConfig, form)
     : transformForSubmit(formConfig, form);
 
-  const version = form.data.hlrV2 ? '1' : '0';
+  const version = '1';
```

Comment:
Same thing here, in my eyes, one less var the better.

Reply:
Yep, I'll change this too!